test/test_integration_resultset.rb

```diff
@@ -139,7 +146,7 @@ def test_close
     assert_predicate stmt, :closed?
     assert_raise(SQLite3::Exception) { result.reset }
     assert_raise(SQLite3::Exception) { result.next }
-    assert_raise(SQLite3::Exception) { result.each }
+    assert_raise(SQLite3::Exception) { result.each.next }
```

Comment:
Why not also add an assertion for `each_hash.next` while we're here?

Reply:
done